### DISCUSSION TITLE
Step by step documentation on how to enable TLS for a wordpress site

### DISCUSSION THREAD

**Discussion by kishaningithub** [coollabsio/coolify#2045] [Q&A] [answered]
I deployed a wordpress site using the standard template on coolify. Would like to know how i can enable TLS?

In the past i have used Caddy server to perform automatic TLS. I see this project uses, trafik for load balancing. I am unsure of its TLS termination capabilities as such. A documentation or a how to guide would really help.

Thanks in advance!

**Comment by peaklabs-dev** [comment, marked as answer]
Just add https:// for the WordPress url. SSL will be done automatically. Support for custom SSL certificates and force https for services may be added in the future.
